sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -86,22 +86,24 @@ public partial class GetAnomaliesForDetectionConfigurationOptions
         public int? Skip { get { throw null; } set { } }
         public System.DateTimeOffset StartTime { get { throw null; } }
     }
-    public partial class GetDatasourceCredentialsOptions
+    public partial class GetAnomalyDimensionValuesOptions
```

Comment:
API moved some types around because we renamed two options types (GetValuesOfDimensionWithAnomaliesOptions => GetAnomalyDimensionValues, GetDimensionValuesOptions => GetMetricDimensionValuesOptions).